### PR TITLE
Fixed many parsing errors that occurred due to small timeout, also some code beautifying

### PR DESCRIPTION
**Pull request by idanpl**
In the commit:
1. timeout was changed from 7 to 60. Problem was that many articles could not be fetched in this time (especially when you have mutlitasking, and I believe that the timeout is starting when the task is created and before the actual request is made // observation....I'm not a python ninja).
2. find_lot_size method is redundant, so I actually removed it.
3. some comment update.

### CODE REVIEW

**Comment by CyberPunkMetalHead** [comment]
I have not been able to replicate your issue, can you temporarily remove the try: and except: conditions and check what error is your code returning?

**Comment by CyberPunkMetalHead** [comment]
Also, please elaborate why find lot size is redundant :)

File: news-analysis.py [hunk 1]
```diff
@@ -93,7 +93,7 @@
 MINUMUM_ARTICLES = 1
 
 # define how often to run the code (check for new + try to place trades)
-# in minutes
+# in seconds
```
Comment (CyberPunkMetalHead) [inline news-analysis.py:96]: This should be in minutes, as you're multiplying this by 60 in the execution of the code

File: news-analysis.py [hunk 2]
```diff
@@ -130,29 +130,9 @@ def ticker_socket(msg):
     conn_key = bsm.start_symbol_ticker_socket(coin+PAIRING, ticker_socket)
 bsm.start()
 
-
-def find_lot_size():
```
Comment (CyberPunkMetalHead) [inline news-analysis.py:134]: Why has this been removed and how do you account for different step sized in order? ie: BTC is 0.000001 and XRP is 0.1. The incorrect format volume will raise an exception and fail to place the order.

**Review by CyberPunkMetalHead — CHANGES_REQUESTED** [review]
REPEAT_EVERY should be in minutes, not seconds
find_lot_size is used to determine how many decimal points of accuracy in the order volume. Orders fail without the proper format.
I am unable to replicate the issue when the timeout is 7 seconds, can you execute the code outside of the try-except block?

File: news-analysis.py (hunk 1 above)
Comment (idanpl) [reply news-analysis.py:96]: it was changed to 1....in line 473....now it's back to 60 after your commit :-)

Comment (idanpl) [reply news-analysis.py:96]: Re the first issue, it was also reported by another user. Not only on my machine...perhaps a timeout that we received due to slower network.

File: news-analysis.py (hunk 2 above)
Comment (idanpl) [reply news-analysis.py:134]: This was fixed in commit 394598e by rybens92.

**Comment by idanpl** [comment]
With all the changes made after the pull request, all that remains is the timeout. Will simply close this pull request.

**Pull request closed without merging**